services/QuillLMS/app/controllers/teachers/classrooms_controller.rb

```diff
@@ -60,8 +60,7 @@ def create_students
   def remove_students
     StudentsClassrooms
       .where(student_id: params[:student_ids], classroom_id: params[:classroom_id])
-      .archive_all
```

Comment:
Good catch on this one.  I learned something new.  https://stackoverflow.com/a/39646223

I noticed we use `update_all` in many places so there might be other places where this lack of callbacks might happen.